dynacred/src/Data.ts

```diff
@@ -297,7 +300,7 @@ export class Data {
      * fields for initialization of the class.
      * @param obj (optional) object with all fields for the class.
      */
-    constructor(obj: any = {}) {
+    constructor(readonly config: Config, obj: any = {}) {
```

Comment:
Giving the `Config` here doesn't really make sense. I can think only of the two following use-cases:
- `Data.load()` finds data in the browser -> config not needed
- `Data.connectByzcoin()` needs to setup `bc` -> pass config (or better, ByzCoinRPC) there